### ISSUE TITLE
[Bug] Connected Textures not working with Canvas Renderer

### ISSUE THREAD

**Issue by scottycool64** [SuperMartijn642/Rechiseled#58]
**Version Info**
- Minecraft, 1.19.4
- Rechiseled, 1.1.4a

**Description of the Bug**
Textures do not connect, and under certain cases bug out.

**Steps to Reproduce**
place various blocks

**Screenshots**
...
![2023-10-16_21 25 04](https://github.com/SuperMartijn642/Rechiseled/assets/73369785/d9f50a40-9c82-479d-8200-ee59e11d4a32)

**Comment by SuperMartijn642** [comment]
Please share what other mods you are using

**Comment by scottycool64** [comment]
the only mods i could think to be related:
Canvas Renderer
Continuity
owo-lib
Fusion
<details>
<summary>full list: </summary>

Advanced Netherite
AppleSkin
AttributeFix
Bookshelf
Canvas Renderer
Cardinal Components API
CC: Tweaked
Charm of Undying
Chimes
Cloth Config API
Collective
Continuity
Creeper Safety
Customizable Elytra
Customizable Player Models
Customizable Player Models Simple Voice Chat compat
Diamond Chest Shops
Diamond Economy
Enchantment Descriptions
Fabric API
Farmer's Delight
Fusion (connected textures)
Just Enough Items
Macaw's Bridges
Macaw's Doors
Macaw's Fences and Walls
Macaw's Roofs
Macaw's Trapdoors
Macaw's Windows
MmmMmmMmmMmm
Moonlight Lib
No Chat Reports
owo-lib
Patchouli
Pehkui
Reacharound
Rechiseled
Redstone Pen
Runelic
Simple Voice Chat
Simply Light
Starter Kit
SuperMartijn642's Config Lib
SuperMartijn642's Core Lib
Trinkets
Twigs
Victus
WorldEdit

</details>

**Comment by SuperMartijn642** [comment]
> the only mods i could think to be related:
Canvas Renderer

You were correct, seems it is caused by Canvas Renderer. As far as I can tell, the quad texture coordinates are correct, Canvas is just somehow refusing to render them correctly. Please report the issue to them.

(Even outside of that, just from loading the game with Canvas Renderer and messing around in the world a bit, it seems to break some vanilla stuff as well and just generally seems to cause quite a few bugs.)

**Comment by scottycool64** [comment]
i haven't noticed that many bugs with canvas renderer, in fact to me it looks even better considering its little bundled improvements. Plus, for the mod i'm making i'm going to stretch it as far as it can go to do what i want it to

**Comment by SuperMartijn642** [comment]
Might be that I just got unlucky, but just some of the things I encountered were weird artifacts when resizing the game window, hitboxes from F3+B being very large, the chunk grid from F3+G moving and rotating with the camera, some items being very dark, and many models randomly breaking after reloading resource packs with F3+T.

**Comment by scottycool64** [comment]
idk, looks perfect to me. Now that you say it, i just tested the chunk grid, and yeah that's a bit strange. Definitely adding that in the issue i put. Never used F3+T either.

**Comment by scottycool64** [comment]
yeah, looks like there are a LOT of bugs on their issue tracker. Probably gonna be a solid minute before they get around to any of this.
